app.test.js

```diff
@@ -127,3 +126,50 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("get all comments for an article", () => {
+    test("200: sends an array of comments belonging to a single article to the client", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.comments).toEqual(expect.any(Array));
+          expect(Object.keys(response.body.comments[0])).toEqual(
+            expect.arrayContaining([
+              "comment_id",
+              "votes",
+              "created_at",
+              "author",
+              "body",
+              "article_id"
+            ])
+          );
+          response.body.comments.forEach((comment) => {
+            expect(comment.article_id).toBe(1);
+          });
+        });
+    });
+    test("200: article comments should be served in an array with the most recent comments first", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const { comments } = response.body;
+          expect(Array.isArray(comments)).toBe(true);
```

Comment:
A .length check would be better than an .isArray here. An empty array would pass your toBeSortedByTest, as there's nothing to compare. Checking the length will ensure that an empty array won't pass!